### ISSUE TITLE
Updated Grafana dashboards

### ISSUE THREAD

**Issue by riptl** [terorie/discord-influx#9]
The existing Grafana dashboards broke with version 1.0 of this bot.

TODO:
* Create a new Grafana dashboard and upload it to this repo.
* Upload dashboard to Grafana cloud.
* Update README instructions.

**Comment by Jcodeerd** [comment]
Any updates?

**Comment by riptl** [comment]
@Jcodeerd Feel free to work on this issue.

**Comment by Jcodeerd** [comment]
Yeah, I just recently started with grafana, and databases, so I don't really have the experience to do so 😅

**Comment by riptl** [comment]
InfluxDB is one of the easier data sources to add into Grafana, so with just some online guide you should get quite far.
I don't have any free time to put towards this repo, so I'm archiving it for now.
Feel free to fork if you want to pick up the existing code base.
Thanks for checking it out though!
